### ISSUE TITLE
Is there any validation set or test set used?

### ISSUE THREAD

**Issue by Bayeees** [brownvc/R3GAN#14]

**Comment by nick-bvc** [comment]
No, generative models are not typically evaluated on some held-out validation/test set. It's always the generated samples against the training set.
